lib/src/main/java/fi/epicbot/toster/model/ReportConfig.kt

```diff
@@ -2,8 +2,15 @@ package fi.epicbot.toster.model
 
 import fi.epicbot.toster.report.DefaultReporter
 import fi.epicbot.toster.report.Reporter
+import fi.epicbot.toster.report.formatter.JsonFormatter
 
 class ReportConfig(
     var enable: Boolean = true,
-    var reporters: MutableList<Reporter> = mutableListOf(DefaultReporter()),
+    var reporters: MutableList<Reporter> = mutableListOf(
```

Comment:
Maybe we should keep posibility to set Reporter in config, not from outside?

Reply:
leave such reporters only as user custom reporters and add default only when use them all and pass it as extra parameter?